.gitignore

```diff
@@ -12,3 +12,4 @@ geckodriver.log
 debug.log
 __pycache__
 venv*
+
```

Comment:
Any reason you have introduced an extra line here?

Reply:
Fixed it